kratos/utilities/variable_utils.cpp

```diff
@@ -90,24 +80,14 @@ array_1d<double, 3> VariableUtils::SumConditionVectorVariable(
 {
     KRATOS_TRY
 
-    array_1d<double, 3> sum_value = ZeroVector(3);
-    const auto& r_comm = rModelPart.GetCommunicator();
-
-    #pragma omp parallel
-    {
-        array_1d<double, 3> private_sum_value = ZeroVector(3);
+    using ReductionType = typename std::conditional< std::is_scalar<array_1d<double, 3>>::value , SumReduction<double> , Array3Reduction >::type;
```

Comment:
Same.

Reply:
no can also be `double`

Reply:
ah no you are right!
Here it is always `array_1d`!

@ashishdarekar then you can directly use the `Array3Reduction` here.

Reply:
> no can also be `double`

(we should probably template the method...)

Reply:
thats my plan ;)